### PR TITLE
add FlushInterval param to loadplugin config

### PR DESCRIPTION
**Pull request by zoojar**

### CODE REVIEW

File: manifests/plugin.pp [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-  $order    = '10',
-  $globals  = false,
-  $interval = undef,
-  $plugin   = $name
+  $ensure        = 'present',
+  $content       = undef,
+  $order         = '10',
+  $globals       = false,
+  $interval      = undef,
+  $flushinterval = undef,
```
Comment (bastelfreak) [inline manifests/plugin.pp]: can you please add a datatype to the new parameter?

File: templates/loadplugin.conf.erb [hunk 2]
```diff
@@ -2,6 +2,9 @@
 <% if scope.lookupvar('collectd::collectd_version_real') and (scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '4.9.4']) >= 0) -%>
 <LoadPlugin <%= @plugin %>>
   Globals <%= @globals %>
+<% if @flushinterval and scope.lookupvar('collectd::collectd_version_real') and (scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.2']) >= 0) -%>
+  FlushInterval <%= @flushinterval %>
```
Comment (bastelfreak) [inline templates/loadplugin.conf.erb]: please add a unit test to validate that the condition works

Comment (zoojar) [reply templates/loadplugin.conf.erb]: done!

File: manifests/plugin.pp (hunk 1 above)
Comment (zoojar) [reply manifests/plugin.pp]: done !

File: spec/acceptance/class_plugin_python_spec.rb [hunk 3]
```diff
@@ -57,7 +57,7 @@
       # Run it twice and test for idempotency
       apply_manifest(pp, catch_failures: true)
       apply_manifest(pp, catch_changes: true)
-      shell('sleep 10')
+      shell('sleep 30')
```
Comment (bastelfreak) [inline spec/acceptance/class_plugin_python_spec.rb]: why did you need to increase this? Does collectd now start slower?

**Comment by zoojar** [comment]
To try to get the tests to pass. Wondered if it was some kind of race
condition

File: spec/acceptance/class_plugin_python_spec.rb (hunk 3 above)
Comment (zoojar) [reply spec/acceptance/class_plugin_python_spec.rb]: To try to get the tests to pass. Wondered if it was some kind of race
condition
